assets/js/googlesitekit/datastore/site/urls.js

```diff
@@ -104,6 +104,70 @@ export const selectors = {
 		} );
 	} ),
 
+	/**
+	 * Gets the Site Kit documentation URL.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @param {string} slug  The slug of the documentation page.
+	 * @return {string} The Site Kit support URL.
+	 */
+	getDocumentationLinkURL: createRegistrySelector(
+		( select ) => ( state, slug ) => {
+			const proxySupportLink = select(
+				CORE_SITE
+			).getProxySupportLinkURL();
+
+			if ( ! slug ) {
+				return proxySupportLink;
+			}
+
+			return `${ proxySupportLink }?doc=${ encodeURIComponent( slug ) }`;
+		}
+	),
+
+	/**
+	 * Gets the relevant troubleshooting URL.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @param {Object} error The error object.
+	 * @return {string} The Site Kit support URL.
+	 */
+	getErrorTroubleshootingLinkURL: createRegistrySelector(
+		( select ) => ( state, error ) => {
+			function isNumber( n ) {
+				return ! isNaN( parseFloat( n ) ) && ! isNaN( n - 0 );
+			}
```

Comment:
Please use the `isNumber` method from `lodash`. As a side note for future reference, in the case of creating a helper function, make it outside of the scope as an independent function.